config/ember-try.js

```diff
@@ -21,6 +26,11 @@ module.exports = async function () {
             'ember-source': '3.27.0-beta.1',
           },
         },
+        env: {
+          EMBER_OPTIONAL_FEATURES: JSON.stringify({
+            'jquery-integration': true,
```

Comment:
not related to reproduction. I've added it in a commit after the bug was reproduced, just to confirm the flag does not have any impact.